building/build_from_commandline.adoc

```diff
@@ -0,0 +1,30 @@
+#  Building locally via commandline
+
+First, fetch the sources of the project you want to build from Github. For details, see link:configuring_git_workflow.adoc#Sources[Configuring Git Workflow - Sources].
+
+Then, switch into the directory where you checked out the sources, and run this:
+
+
+[source,bash]
+----
+cd ~/my-github-stuff/jbosstools-base
+
+# build everything and run all tests
```

Comment:
I've added this note, because -fae and -e are very useful and should be encouraged... but only for the correct use cases:

Setting your build to only fail at the end with `-fae` is not recommended when building locally for development, as you will have to wait to the end of the whole maven reactor to see errors. 

Use of `-fae` *is* recommended for CI builds, multimodule project, or for collecting all errors instead of stopping after the first one.

Use of `-e` is recommended if you want to see a stack trace should something fail.